### PR TITLE
S3 t1start

### PR DESCRIPTION
**Pull request by bentleybiscuit**

### CODE REVIEW

File: functions/gameStarted.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import {game} from 'main.js'
+//create eventListener for New Game html element with params of game object and shuffled array
+document.querySelector('.newGame').addEventListener('click', function (e) {
+    e.stopImmediatePropagation()
+    if (game.gameStarted = true) {
+        document.querySelector('.cards').styles.backgroundImage="url('images/cardBack.jpg')"
+    } else {
+        game.gameStarted = true
+        var shuffledDeck = shuffle()
```
Comment (sudokufan) [inline functions/gameStarted.js]:
```suggestion
        game.cardArray = shuffle(game.cardArray)
```

File: functions/gameStarted.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+        return shuffledDeck
```
Comment (sudokufan) [inline functions/gameStarted.js]:
```suggestion

```

File: functions/.idea/.gitignore [hunk 3]
```diff
@@ -0,0 +1,3 @@
+
```
Comment (sudokufan) [inline functions/.idea/.gitignore]: .idea/.gitignore is the branch at the moment, needs to be moved to hide the .idea files

File: main.js [hunk 4]
```diff
@@ -1,3 +1,8 @@
+export let game = {
```
Comment (sudokufan) [inline main.js]:
```suggestion
let game = {
```

File: main.js [hunk 5]
```diff
@@ -1,3 +1,8 @@
...
+    gameStarted: false,
+    cardArray: [1, 1, 2, 2, 3, 3, 4, 4, 5, 5, 6, 6, 7, 7, 8, 8]
+}
+
```
Comment (sudokufan) [inline main.js]:
```suggestion
export {game}
```

File: functions/gameStarted.js [hunk 6]
```diff
@@ -0,0 +1,12 @@
+import {game} from 'main.js'
```
Comment (sudokufan) [inline functions/gameStarted.js]:
```suggestion
import {game} from '../main.js'
```

File: functions/gameStarted.js [hunk 7]
```diff
@@ -0,0 +1,11 @@
+import {game} from '../main.js'
```
Comment (bentleybiscuit) [inline functions/gameStarted.js]:
```suggestion
import {game} from '../main.js'
import 'shuffle.js'
```

**Pull request merged**